### PR TITLE
Support more than one namespace for policies

### PR DESCRIPTION
**Pull request by simonbaird**
This grew a little from the original version, but the main intent is the same, i.e. support running different policies when verifying different things, and get ready to do that for verifying a pipeline definition.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -172,9 +172,12 @@ CONFIG_DATA_FILE=$(DATA_DIR)/config.json
 INPUT_DIR=./input
 INPUT_FILE=$(INPUT_DIR)/input.json
 
+RELEASE_NAMESPACE=release.main
+
 POLICIES_DIR=./policies
 OPA_FORMAT=pretty
-OPA_QUERY=data.main.deny
+OPA_QUERY=data.$(RELEASE_NAMESPACE).deny
```
Comment (joejstuart) [inline Makefile:179]: With conftest you can just specify the namespace. This might only select the deny rules, I'm not sure though.

**Comment by robnester-rh** [comment]
Wasn't there discussion around renaming the `policies` folder to `policy` as it's the "standard" name expected?

**Review by robnester-rh — APPROVED** [review]

File: policies/pipeline/basic.rego [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import data.lib
+
+expected_kind := "Pipeline"
+
+# (Not sure if we need this, but I'm using it to test the docs build.)
+
+# Fixme: It doesn't fail if the kind key is entirely missing..
+
+# METADATA
+# title: Check the kind is "PipelineRun"
```
Comment (zregvart) [inline policies/pipeline/basic.rego:12]:
```suggestion
# title: Check the kind is "Pipeline"
```

**Review by zregvart — APPROVED** [review]
Looks straightforward

File: docs/index.md [hunk 3]
```diff
@@ -13,8 +13,21 @@ Those policies are defined using
...
 described here.
 
-Policy Rules
-------------
+Pipeline Policy
+----------------
+
+### Basic Rules
+
+#### `[unexpected_kind]` Check the kind is "PipelineRun"
```
Comment (robnester-rh) [inline docs/index.md:21]: Should this be `"Pipeline"` instead of `"PipelineRun"`?

File: policies/pipeline/basic.rego (hunk 2 above)
Comment (robnester-rh) [reply policies/pipeline/basic.rego:12]: +1 on this suggested change, see associated comment in `index.md`

**Review by robnester-rh — COMMENTED** [review]
Pending until all commits are finalized. Notes left.

File: docs/index.md (hunk 3 above)
Comment (simonbaird) [reply docs/index.md:21]: Oh yeah, thanks!

**Review by joejstuart — APPROVED** [review]
Looks good. Do we have to coordinate merging with an update to `--policy` in build-definitions? https://github.com/redhat-appstudio/build-definitions/blob/main/appstudio-utils/util-scripts/check-ec-policy.sh#L20

**Comment by simonbaird** [comment]
I want to merge, but first we need to:
* Pin the sha in e2e
* Get a MR ready for the EC task (as @joejstuart suggests above)

**Comment by simonbaird** [comment]
Since https://github.com/redhat-appstudio/e2e-tests/pull/100 is merged, I think we can merge this without breaking e2e. So here goes...

**Pull request merged**